src/lib/Scenes/Collection/Screens/CollectionArtworkPreview.tsx

```diff
@@ -0,0 +1,40 @@
+import { Separator } from "@artsy/palette"
+import { CollectionArtworkPreview_collection } from "__generated__/CollectionArtworkPreview_collection.graphql"
+import GenericGrid from "lib/Components/ArtworkGrids/GenericGrid"
+import React from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+
+interface Props {
+  collection: CollectionArtworkPreview_collection
+}
+
+export class CollectionArtworkPreview extends React.Component<Props> {
+  render() {
+    const { collection } = this.props
+    if (!collection) {
+      return null
+    }
+    const artworks = collection.artworks.edges.map(({ node }) => node)
+
+    return (
+      <>
+        <Separator mb={4} />
+        <GenericGrid artworks={artworks} />
+      </>
+    )
+  }
+}
+
+export const CollectionArtworkPreviewContainer = createFragmentContainer(CollectionArtworkPreview, {
+  collection: graphql`
+    fragment CollectionArtworkPreview_collection on MarketingCollection {
+      artworks: artworksConnection(sort: "-merchandisability", first: 6) {
```

Comment:
Should be `-decayed_merch` _n'est-ce pas_? Slack convo [here](https://artsy.slack.com/archives/C9SATFLUU/p1576685422107800)...

Reply:
Yes, but I want to follow up with this in a sep PR bc it requires a change in Reaction and Emission and for now I think the data should be consistent.